pkg/cluster/version.go

```diff
@@ -28,6 +28,24 @@ type openShiftClusterDocumentVersionerService struct{}
 func (service *openShiftClusterDocumentVersionerService) Get(ctx context.Context, doc *api.OpenShiftClusterDocument, dbOpenShiftVersions database.OpenShiftVersions, env env.Interface, installViaHive bool) (*api.OpenShiftVersion, error) {
 	requestedInstallVersion := doc.OpenShiftCluster.Properties.ClusterProfile.Version
 
+	// add the default OCP version as we require it as an install target
+	// if this is removed, we need to also update the logic in
+	// pkg/frontend/frontend.go, pkg/frontend/validate.go
+	defaultVersion := &api.OpenShiftVersion{
+		Properties: api.OpenShiftVersionProperties{
+			Version:           version.InstallStream.Version.String(),
+			OpenShiftPullspec: version.InstallStream.PullSpec,
+			InstallerPullspec: fmt.Sprintf("%s/aro-installer:release-%s", env.ACRDomain(), version.InstallStream.Version.MinorVersion()),
```

Comment:
Also wanted to highlight that by using this hard-coded InstallerPullspec, together with dropping the LiveConfig override, we are losing the ability to set a different value for the default.

Reply:
Fixed by adding the installer spec override at the top of the function.